### ISSUE TITLE
No Batchnorm in A Simple NN Like Lenet

### ISSUE THREAD

**Issue by zhangxj19** [onnx/onnx-mlir#1225]
Here is my simple version of Lenet which is composed of 1 conv layer and 1 linear layer.
There is a batchnorm right after the conv.
![image](https://user-images.githubusercontent.com/32873170/156962425-1f7b4c77-8a54-4ca2-a2f3-b6f6f289d6e6.png)
But there is no batchnorm after the conv even I turn off the conv-bn-fuse pass (Actually I turned off all the passes)
![image](https://user-images.githubusercontent.com/32873170/156962369-fb4973d3-4720-4287-ad86-d2be358e0966.png)

**Comment by tungld** [comment]
Could you check if your .onnx model really contains a batchnorm or not please? You can open a .onnx model by using https://netron.app/.

**Comment by zhangxj19** [comment]
You are right. There is no batchnorm in onnx model.
![image](https://user-images.githubusercontent.com/32873170/157194619-b49b86b1-0a89-4c08-beec-88628d355465.png)
It's weird since there are indeed batchnorm in pytorch model.
![image](https://user-images.githubusercontent.com/32873170/157195028-5f7758a4-bbfc-4402-8dd2-edb62173d616.png)
Here is the code How the pytorch model is export to onnx.
![image](https://user-images.githubusercontent.com/32873170/157195081-2da4fcb4-54f7-4d63-acff-6fefc101d7f7.png)

**Comment by zhangxj19** [comment]
> You are right. There is no batchnorm in onnx model. ![image](https://user-images.githubusercontent.com/32873170/157194619-b49b86b1-0a89-4c08-beec-88628d355465.png) It's weird since there are indeed batchnorm in pytorch model. ![image](https://user-images.githubusercontent.com/32873170/157195028-5f7758a4-bbfc-4402-8dd2-edb62173d616.png) Here is the code How the pytorch model is export to onnx. ![image](https://user-images.githubusercontent.com/32873170/157195081-2da4fcb4-54f7-4d63-acff-6fefc101d7f7.png)

I found the solution below
https://github.com/pytorch/pytorch/issues/49226
Thank you for answering!
